wishlist-backend/products/views.py

```diff
@@ -4,13 +4,14 @@
 from .serializers import ProductSerializer
 
 def get_products(request,category=None):
+
```

Comment:
only add new line if you will add the method doc like:
"""
parameters and return description ... 
"""

Reply:
Got it. What I have understood from this is to only insert a new line if I am writing a docstring for the function.